src/shared_gui_components/MeasureManager.cpp

```diff
@@ -945,18 +945,43 @@ bool MeasureManager::checkForUpdates(const openstudio::path& measureDir, bool fo
   return result;
 }
 
+void MeasureManager::checkForRemoteBCLUpdates() {
+  RemoteBCL remoteBCL;
+  int numUpdates = remoteBCL.checkForMeasureUpdates();
+  if (numUpdates == 0) {
+    QMessageBox::information(m_app->mainWidget(), tr("Measures Updated"), tr("All measures are up-to-date."));
+  } else {
+    std::vector<BCLSearchResult> updates = remoteBCL.measuresWithUpdates();
+
+    QString text(QString::number(numUpdates) + tr(" measures have been updated on BCL compared to your local BCL directory.\n")
+                 + tr("Would you like update them?"));
+
+    QString detailedText;
+    for (const BCLSearchResult& update : updates) {
+      detailedText += toQString("* " + update.name() + "\n");
+    }
+    QMessageBox msg(QMessageBox::Question, tr("Measures Updated"), text, QMessageBox::Yes | QMessageBox::No, m_app->mainWidget());
+    msg.setDetailedText(detailedText);
+
+    int result = msg.exec();
+    if (result == QMessageBox::Yes) {
+      remoteBCL.updateMeasures();
+    }
+  }
+}
+
 void MeasureManager::downloadBCLMeasures() {
-  auto remoteBCL = new RemoteBCL();
-  int numUpdates = remoteBCL->checkForMeasureUpdates();
+  RemoteBCL remoteBCL;
+  int numUpdates = remoteBCL.checkForMeasureUpdates();
```

Comment:
Bizarre that we only called this here

Reply:
Also bizarre it's actually never called anywhere. There's a lot of cruft in the codebase, cleaning it up at some point would be nice...